### PR TITLE
Make all pages urls links dynamic with current site

### PR DESCRIPTION
**Pull request by AzmiTouil**

### CODE REVIEW

File: digital-workplace-webapps/src/main/webapp/vue-app/sidebar/mySettings/components/MySettingsApp.vue [hunk 1]
```diff
@@ -16,12 +16,12 @@
...
       <v-row class="mx-0 settingsLinks">
         <v-list class="flex" dense>
-          <v-list-item class="ml-12 px-2" @click="openURI('/portal/dw/account', '_self')" >
+          <v-list-item class="ml-12 px-2" @click="navigateTo('account', '_self')" >
             <v-list-item-avatar size="30" class="mr-3"><i class="securityIcon"></i></v-list-item-avatar>
             <v-list-item-content class="body-2 mySettingsItemLabel">{{ this.$t('homepage.sidebar.settings.account') }}</v-list-item-content>
           </v-list-item>
 
-          <v-list-item class="ml-12 px-2" @click="openURI('/portal/dw/notifications', '_self')" >
+          <v-list-item class="ml-12 px-2" @click="navigateTo('notifications', '_self')" >
```
Comment (azayati) [inline digital-workplace-webapps/src/main/webapp/vue-app/sidebar/mySettings/components/MySettingsApp.vue]: remove _self here

File: digital-workplace-webapps/src/main/webapp/vue-app/sidebar/mySettings/components/MySettingsApp.vue [hunk 2]
```diff
@@ -16,12 +16,12 @@
       </v-row>
       <v-row class="mx-0 settingsLinks">
         <v-list class="flex" dense>
-          <v-list-item class="ml-12 px-2" @click="openURI('/portal/dw/account', '_self')" >
+          <v-list-item class="ml-12 px-2" @click="navigateTo('account', '_self')" >
```
Comment (azayati) [inline digital-workplace-webapps/src/main/webapp/vue-app/sidebar/mySettings/components/MySettingsApp.vue]: remove _self here

Comment (AzmiTouil) [reply digital-workplace-webapps/src/main/webapp/vue-app/sidebar/mySettings/components/MySettingsApp.vue]: Done

File: digital-workplace-webapps/src/main/webapp/vue-app/sidebar/mySettings/components/MySettingsApp.vue (hunk 1 above)
Comment (AzmiTouil) [reply digital-workplace-webapps/src/main/webapp/vue-app/sidebar/mySettings/components/MySettingsApp.vue]: Done

**Pull request merged**